pom.xml

```diff
@@ -5,7 +5,7 @@
   <artifactId>sgv2-jsonapi</artifactId>
   <version>1.0.26-SNAPSHOT</version>
   <properties>
-    <stargate.version>2.1.0-BETA-25</stargate.version>
+    <stargate.version>2.1.0-BETA-24</stargate.version>
```

Comment:
Need to be careful not to merge this change accidentally.

Reply:
thx, not sure how  I would have overridden a change from main here